lib/src/migrator.dart

```diff
@@ -55,6 +63,16 @@ abstract class Migrator extends Command<Map<Uri, String>> {
         allMigrated[file] = migrated[file];
       }
     }
+    if (missingDependencies.isNotEmpty) {
+      var count = missingDependencies.length;
+      emitWarning(
+          "$count dependenc${count == 1 ? 'y' : 'ies'} could not be found.");
```

Comment:
It would be good to list the URLs here. It's a little more verbose for users who genuinely don't care, but it'll serve as a sanity check that it's the sort of URLs they're expecting.